src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -846,15 +846,15 @@ private static bool EquivalentToInternal(GreenNode node1, GreenNode node2)
                 case 1:
                     return select(list[0]);
                 case 2:
-                    return SyntaxList.List(select(list[0]), select(list[1]));
+                    return Syntax.InternalSyntax.SyntaxList.List(select(list[0]), select(list[1]));
```

Comment:
had to qualify since there is now a public type called SyntaxList (which contains the factory method).